app/models/user.rb

```diff
@@ -87,8 +87,17 @@ def archive!
   def self.from_omniauth(auth)
     user = where(provider: auth.provider, uid: auth.uid).first_or_create do |u|
       u.name = auth[:info][:name]
+      u.email = auth[:info][:email]
```

Comment:
These attributes were only added when synchronizing users, not when logging in via OAuth